client/components/Timer/timer.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+
+type TimerProps = {
+  time: number;
+  onEnd?: () => void;
+}
+
+function formatTime(time: number): string {
```

Comment:
Zamieniłbym to na funkcje strzałkową jako że wszędzie w kodzie mamy funkcje strzałkowe

Reply:
ok :)